src/Panel.js

```diff
@@ -23,6 +25,18 @@ const Panel = React.createClass({
     };
   },
 
+  getInitialState(){
+    let defaultExpanded = this.props.defaultExpanded != null ?
+      this.props.defaultExpanded :
+        this.props.expanded != null ?
+        this.props.expanded :
```

Comment:
but this...
Isn't it an anti-pattern here http://facebook.github.io/react/tips/props-in-getInitialState-as-anti-pattern.html ?